port_ocean/core/handlers/webhook/processor_manager.py

```diff
@@ -72,17 +70,29 @@ async def process_queue(self, path: str) -> None:
         """Process events for a specific path in order"""
         while True:
             matching_processors: list[AbstractWebhookProcessor] = []
-            event: WebhookEvent | None = None
+            webhookEvent: WebhookEvent | None = None
             try:
-                event = await self._event_queues[path].get()
-                with logger.contextualize(webhook_path=path, trace_id=event.trace_id):
-                    matching_processors = self._extract_matching_processors(event, path)
-                    await asyncio.gather(
-                        *(
-                            self._process_single_event(processor, path)
-                            for processor in matching_processors
+                queue = self._event_queues[path]
+                webhookEvent, eventContext = await queue.get()
+                with logger.contextualize(
+                    webhook_path=path, trace_id=webhookEvent.trace_id
+                ):
+                    async with event_context(
+                        EventType.LIVE_EVENT,
+                        trigger_type="machine",
+                        parent_override=eventContext,
+                    ):
```

Comment:
general thought, I think we need to be consistent, with whether we are naming it live events or webhook ( I lean towards webhook, but open to suggestions )